### ISSUE TITLE
AGM air temperature?

### ISSUE THREAD

**Issue by Tachii** [KoffeinFlummi/AGM_Compatibility#23]
I've noticed that you only have those values on vanilla islands. Does that mean it wont work on any other custom island? Maybe add this as well for custom islands? I can do it if it's necessary.

And how does this Temp parameters work?
What does Mean and Amp parameters stand for?
```
class CfgWorlds {
  class CAWorld;

  class Stratis: CAWorld {
    AGM_TempMeanJan = 7.4;
    AGM_TempMeanJul = 25.9;
    AGM_TempAmplitudeJan = 6.4;
    AGM_TempAmplitudeJul = 9.2;
  };

  class Altis: CAWorld {
    AGM_TempMeanJan = 7.4;
    AGM_TempMeanJul = 25.9;
    AGM_TempAmplitudeJan = 6.4;
    AGM_TempAmplitudeJul = 9.2;
  };
};

```

**Comment by Tachii** [comment]
hello?

**Comment by esteldunedain** [comment]
hello :) 

Yeah, those values are there for any mod that wants to be super realistic. But if they are not configured for a certain map, then they are estimated from the latitude of the map. That estimation was adjusted to fit a variety of places and works rather well (https://github.com/KoffeinFlummi/AGM/pull/1039#issuecomment-53469973).

```
    AGM_TempMeanJan = Mean Temperature on January (ºC)
    AGM_TempMeanJul  = Mean Temperature on July (ºC)
    AGM_TempAmplitudeJan = Temperature Amplitude on January (ºC);
    AGM_TempAmplitudeJul = Temperature Amplitude on July (ºC);
```

`Amplitude` is the diference between maximum and minimum hourly temperature for those months (e.g. how much hot is during the day that in the night)


TL;DR: it's not that important if those values are not present.

**Comment by Tachii** [comment]
Thanks for the answers, why I asked is because I wanna do a small mod for my community that lowers the overall lvl of stamina if temperature of air for example is above 20c and so on.

**Comment by esteldunedain** [comment]
BTW, @Tachii, if case you want to check the current temperature in-game, the Kestrel reports that ("AGM_Kestrel").

**Comment by Tachii** [comment]
Yeah, thanks, but i'm using an agm function insted.
Something like this:

```
if (hasInterface) then
{
	[] spawn
	{
	if (player != player) then
		{
			waitUntil { player == player };
		};
		if !(player in allUnits) then
		{
			waitUntil { player in allUnits };
		};
		
		while {true} do
		{ 
			sleep 5;
			if ((getDammage player == 0) && (getFatigue player < 0.3)) then
			{
				player setFatigue 0.3;
			};
			if ((getDammage player == 0) && (getFatigue player < 0.3)) then
			{
				player setFatigue 0.3;
			};
		};
	};
};
```
